api/bag_layer.cpp

```diff
@@ -0,0 +1,203 @@
+
+#include "bag_layer.h"
+#include "bag_metadata.h"
+#include "bag_private.h"
+#include "bag_trackinglist.h"
+
+#include <array>
+#include <h5cpp.h>
```

Comment:
it should be "H5Cpp.h" for portability

Reply:
Thanks, yes.  I have some local modifications containing this change.  It definitely needs to be the proper case for our non-Windows users.